src/Raven.Server/Json/WebsocketStream.cs

```diff
@@ -51,17 +46,23 @@ public override long Length
         public override void Write(byte[] buffer, int offset, int count)
         {
             ThrowOnDisposed();
-
             var sendTask = _webSocket.SendAsync(new ArraySegment<byte>(buffer), 
                                                 WebSocketMessageType.Text, 
                                                 false,_cancellationToken);
-
             _activeWriteTasks.Add(sendTask);
-            sendTask.ContinueWith(t => _activeWriteTasks.TryRemove(t), _cancellationToken);
+            try
+            {
+                sendTask.ContinueWith(t => _activeWriteTasks.TryRemove(t), _cancellationToken);
+            }
+            catch (ObjectDisposedException)
```

Comment:
Does NOT do what you think it does, and it is wrong, to boot.
Do not mix sync & async code here, at all